widgetssdk/src/main/java/com/glia/widgets/chat/ChatManager.kt

```diff
@@ -50,40 +50,49 @@ internal class ChatManager constructor(
         onQuickReplyReceived: (List<GvaButton>) -> Unit,
         onOperatorMessageReceived: (count: Int) -> Unit
     ): Flowable<List<ChatItem>> {
-        compositeDisposable.apply {
-            clear()
-            addAll(subscribeToMessages(onHistoryLoaded, onOperatorMessageReceived), subscribeToQuickReplies(onQuickReplyReceived))
-        }
 
-        return state.map(State::immutableChatItems).distinctUntilChanged().onBackpressureLatest().share()
+        extracted(onHistoryLoaded, onOperatorMessageReceived, onQuickReplyReceived)
+
+        return state.map(State::immutableChatItems).onBackpressureLatest().share()
     }
 
-    fun stop() = compositeDisposable.clear()
+    private fun extracted(
```

Comment:
It's not critical - I think it's better to name this fun `onHistoryExtracted()` or any other name with a verb. `extracted` is in past tense and it's not clear what is extracted and what the function does.

Reply:
Thanks, this is a leftover, at some point I extracted some part of the code to the method and forgot to rename it.

Reply:
fixed - https://github.com/salemove/android-sdk-widgets/commit/fa506584650c3ca6870b22f4570256ea193ea018